source/Halibut.Tests/FailureModesFixture.cs

```diff
@@ -109,7 +109,7 @@ public async Task FailWhenServerThrowsDuringADataStream(ClientAndServiceTestCase
         {
             using (var clientAndService = await clientAndServiceTestCase.CreateTestCaseBuilder()
                        .WithStandardServices()
-                       .WithHalibutLoggingLevel(LogLevel.Trace)
+                       .WithHalibutLoggingLevel(LogLevel.Info)
```

Comment:
Now that we've made the default info, do we still want to explicitly set this to 'Info'?

Reply:
No but that can be a cleanup after this PR